decidim-debates/lib/decidim/debates/debate_serializer.rb

```diff
@@ -37,12 +37,17 @@ def serialize
           component: { id: component.id },
           reference: debate.reference,
           comments: debate.comments_count,
-          followers: debate.follows.size,
+          followers_count: debate.followers.size,
```

Comment:
We should use the counter cache here (you need to update the spec too)

```suggestion
          follows_count: debate.follows_count,
```